<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_form</class>
 <widget class="QMainWindow" name="admin_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>admin_form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>961</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>添加界面</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>390</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>110</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>出发时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>出发地</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>航班号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>239</x>
        <y>100</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>到达时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>到达地</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_date">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>16</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2024</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_flightId">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_departureTime">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_departurePlace">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_arrivalPlace">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_arrivalTime">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>经济舱</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>220</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>商务舱</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>头等舱</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>座位数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_baseFare">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_seatAmountOfLvl1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_seatAmountOfLvl3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_seatAmountOfLvl2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_moneyOfLvl2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_moneyOfLvl3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>删除界面</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>航班号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_delFlightId">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>查询界面</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>航班搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>出发地</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>到达地</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>82</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_selectedDeparturePlace">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_selectedArrivalPlace">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_selectedDate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>931</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>修改界面</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>需要修改的航班</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_needToUpdateFlightId">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_needToUpdateDate">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_needToUpdateDepartureTime">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_needToUpdateArrivalTime">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>118</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_needToUpdateDeparturePlace">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_needToUpdateArrivalPlace">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_needToUpdateBaseFare">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>出发时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>出发地</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>到达时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>到达地</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>基础价格</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_addFlight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加航班</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteFlight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除航班</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_searchFlight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查询航班</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改航班</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
